### DISCUSSION TITLE
Google Authenticator (TOTP) in Pocketbase

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#3852] [Show and tell]
Hi all,

Since there is no ability to integrate totp authentication in pocketbase (where many people asked for) I decided to create something in golang. I do not have that much experience in programming, atleast not in golang.. I encouraged myself to start, If you see something that I can improve, please let me know or make a PR.

https://github.com/raizdev/pocketbase-totp

demo: https://qr.starre.dev/sign-in

**Comment by VictorioBerra** [comment]
I am unable to load the repo.
